usr/plugins/devel/caddy/src/opnsense/service/templates/Pischem/Caddy/Caddyfile

```diff
@@ -1,50 +1,71 @@
 # DO NOT EDIT THIS FILE -- OPNsense auto-generated file
 {
-    # Global options
     storage file_system {
         root /usr/local/etc/caddy
     }
-
-    # Global logging
     log {
         output file /var/log/caddy/caddy.log {
             roll_size 100MiB
             roll_keep 5
             roll_keep_for 720h
         }
     }
-
-    # Global TLS settings
-{% if helpers.exists('Pischem.caddy.general.TlsEmail') %}
     email {{ helpers.toList('Pischem.caddy.general.TlsEmail') | first }}
-{% endif %}
-{% if helpers.exists('Pischem.caddy.general.TlsAutoHttps') %}
-    {% set autoHttpsValue = helpers.toList('Pischem.caddy.general.TlsAutoHttps') | first %}
-    {% if autoHttpsValue != "on" %}
-    auto_https {{ autoHttpsValue }}
-    {% endif %}
-{% endif %}
-
-    # Include all .global files from caddy.d directory
+    auto_https {{ helpers.toList('Pischem.caddy.general.TlsAutoHttps') | first }}
```

Comment:
Found a bug here, when auto_https is on, it has to vanish. Right now its being set on "auto_https on" and that prevents caddy from starting because this directive doesn't exist.